### PR TITLE
Added LCM configuration recipes.

### PR DESCRIPTION
**Pull request by Aliasgar16**
In-progress.

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 license 'Apache 2.0'
 description 'Installs/Configures PowerShell on the Windows platform'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '3.1.0'
+version '3.2.0'
```
Comment (NimishaS) [inline metadata.rb]: @adamedx, please comment is the version looks ok.

**Comment by NimishaS** [comment]
`disable_lcm` recipe requires `powershell 5` to be installed. Hence we have included `powershell 5` recipe in `disable_lcm`. 
In case user's current powershell version is less than 5, then powershell 5 is installed and the system gets rebooted. Then user need to run the `disable_lcm` recipe again. We should mention this in `Readme`

**Comment by adamedx** [comment]
@smurawski thoughts on this pr? This is to allow users to configure the lcm.

**Comment by smurawski** [comment]
Thanks!  I grabbed some commits and tweaked a bit of this.  I'll be getting this merged early next week.

**Comment by smurawski** [comment]
Merged as part of #63

**Pull request closed without merging**